assets/src/scss/new-identity/style.scss

```diff
@@ -121,13 +124,16 @@ table.spreadsheet-table.is-color-gp-green {
   }
 }
 
-a:where(.standalone-link) {
+.standalone-link,
+.comment-reply-link {
+  color: var(--link--color);
```

Comment:
Actually I think that we can completely remove that line, they already use the `--link--color` variable